components/form/FormCardDemand.tsx

```diff
@@ -35,19 +35,21 @@ export const FormCardSupply = ({
     color: `#${Math.floor(Math.random() * 16777215).toString(16)}`,
     isEpochDistro: false,
     epochDurationInSeconds: 0,
+    supplyDemandType:'',
     initialEmissionPerSecond: 0,
     emissionReductionPerEpoch: 0,
     CalculationTimeSeries: [],
-    isTemplate: false,
+    isTemplate: true,
```

Comment:
this seems wrong. Templates should only be created via the admin screen and not for every mechanism that is created.

Reply:
did you mean that template should always be false by default unless the admin approves it?

Reply:
Yes. The Mechanism isTemplate field is only set to true in the http://localhost:3000/coreMechanisms screen. All other mechanisms have isTemplate=false.

Reply:
Okay, got the point